src/test/feature/type/ans/int8.ans

```diff
@@ -175,131 +179,131 @@ SELECT '' AS to_char_4, to_char( (q1 * -1), '9999999999999999S'), to_char( (q2 *
            | +456
-           | -4567890123456789
            | +4567890123456789
+           | +123
            | +4567890123456789
+           | -4567890123456789
 (5 rows)
 
 SELECT '' AS to_char_7,  to_char(q2, 'FM9999999999999999THPR') FROM INT8_TBL ;
  to_char_7 |      to_char       
 -----------+--------------------
-           | 123RD
-           | <4567890123456789>
+           | 456TH
            | 4567890123456789TH
+           | 123RD
            | 4567890123456789TH
-           | 456TH
+           | <4567890123456789>
 (5 rows)
 
 SELECT '' AS to_char_8,  to_char(q2, 'SG9999999999999999th')   FROM INT8_TBL ;
  to_char_8 |       to_char       
 -----------+---------------------
-           | +             123rd
-           | -4567890123456789
+           | +             456th
            | +4567890123456789th
+           | +             123rd
            | +4567890123456789th
-           | +             456th
+           | -4567890123456789
 (5 rows)
 
 SELECT '' AS to_char_9,  to_char(q2, '0999999999999999')       FROM INT8_TBL ;
  to_char_9 |      to_char      
 -----------+-------------------
-           |  0000000000000123
            |  0000000000000456
            |  4567890123456789
+           |  0000000000000123
            |  4567890123456789
            | -4567890123456789
 (5 rows)
 
 SELECT '' AS to_char_10, to_char(q2, 'S0999999999999999')      FROM INT8_TBL ;
  to_char_10 |      to_char      
 ------------+-------------------
-            | +0000000000000123
             | +0000000000000456
-            | -4567890123456789
             | +4567890123456789
+            | +0000000000000123
             | +4567890123456789
+            | -4567890123456789
 (5 rows)
 
 SELECT '' AS to_char_11, to_char(q2, 'FM0999999999999999')     FROM INT8_TBL ;
  to_char_11 |      to_char      
 ------------+-------------------
-            | 0000000000000123
             | 0000000000000456
             | 4567890123456789
+            | 0000000000000123
             | 4567890123456789
             | -4567890123456789
 (5 rows)
 
 SELECT '' AS to_char_12, to_char(q2, 'FM9999999999999999.000') FROM INT8_TBL ;
  to_char_12 |        to_char        
 ------------+-----------------------
-            | 123.000
             | 456.000
-            | -4567890123456789.000
             | 4567890123456789.000
+            | 123.000
             | 4567890123456789.000
+            | -4567890123456789.000
 (5 rows)
 
 SELECT '' AS to_char_13, to_char(q2, 'L9999999999999999.000')  FROM INT8_TBL ;
  to_char_13 |        to_char         
 ------------+------------------------
-            |                123.000
-            |                456.000
-            |   4567890123456789.000
-            |   4567890123456789.000
-            |  -4567890123456789.000
+            | $              456.000
+            | $ 4567890123456789.000
+            | $              123.000
+            | $ 4567890123456789.000
+            | $-4567890123456789.000
```

Comment:
I'm leaving the to another JIRA HAWQ-925
Set default locale, timezone & datastyle before running sql command/file.

That is for the whole feature test framework.